### PR TITLE
release notes 2.1.0

### LINKED ISSUE #1009: Release notes
Create release notes for the version tied to Magento 2.3.1

### PR DESCRIPTION
**Pull request by jcalcaben**
## Description

Update the release notes to add a section for the next release

## Related Issue

Closes #1009 

## Motivation and Context

We need to provide release notes for the next release.

## Verification

n/a

## How Has This Been Tested?

n/a

## Screenshots (if appropriate):

n/a

## Proposed Labels for Change Type/Package

Documentation

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have linked an issue to this PR.
- [x] I have indicated the change type and relevant package(s).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All CI checks are green (linting, build/deploy, etc).
- [ ] At least one core contributor has approved this PR.

### CODE REVIEW

**Comment by jcalcaben** [comment]
On hold until the release is ready to go

**Comment by sirugh** [comment]
Per discussion in stand up and earlier in slack, sounds like we want to include all changes from `v2.0.0` to `HEAD` on `develop` (as in, whatever was merged) and then whatever else is in the `2.1.0` milestone that we expect to be merged.

**Comment by jcalcaben** [comment]
Additional items added as requested by @sirugh

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,49 @@
...
+
+Notable changes include:
+
+* Updated GraphQL queries
+* Magento GraphQL query validation tool for buildpack
+
+### Updated GraphQL queries
+
+The Magento 2.3.1 release contains GraphQL schema changes that are not compatible with PWA Studio 2.0.0 presentational components.
+This release adds a mapping layer to the wrapper components to maintain backwards compatibility for the presentational components.
```
[the remark this answers is not in the document]
Comment (jcalcaben) [reply CHANGELOG.md]: note added

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+* [2.0.0](#whats-new-in-200)
+
+## What's new in 2.1.0
+
+Release **2.1.0** is a compatibility release for the core **Magento 2.3.1** release.
+
+Notable changes include:
+
+* Updated GraphQL queries
+* Magento GraphQL query validation tool for buildpack
```
Comment (supernova-at) [inline CHANGELOG.md]: The plugin is in its own directory (and outside of `pwa-buildpack`), but it is still part of the build process (runs when Venia gets built). Should this be updated?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,54 @@
...
+This update also includes a change to the `.env.dist` file in the Venia project.
+This change sets the `MAGENTO_BACKEND_URL` variable to that of a Magento 2.3.1 instance.
+
+If you have previously set up Venia and copied the `.env.dist` file into your project's `.env` file, you must update the `MAGENTO_BACKEND_URL` variable to keep your project compatible.
+
+Pull Request: [#990](https://github.com/magento-research/pwa-studio/pull/990)
+
+### Query validation tool
+
+PWA Studio 2.1.0 creates a new `graphql-cli` plugin called `validate-queries` to replace the `validate-queries.js` script in the Venia package.
```
Comment (supernova-at) [inline CHANGELOG.md]: Slight change to the name of the plugin. Is `validate-magento-pwa-queries` now.

**Pull request merged**